pkg/constants/constants.go

```diff
@@ -10,4 +10,5 @@ const (
 	HierarchyChildren = "hierarchy_children"
 	HierarchyParent   = "hierarchy_parent"
 	HierarchyParents  = "hierarchy_parents"
+	ISO8601Layout     = "2006-01-02T15:04:05Z0700"
```

Comment:
is this the layout conventionally used by other kube types?
https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#Time

Reply:
RFC3339 is the format that is being used by the majority of Kubernetes. 

ISO8601 is also being used in certain places as well

Given that RFC3339 is being used primarily on labels and annotations, I would prefer to move toward that format as we can make use of libraries that are already available to us

@dweebo was the primary contributor to the enhancement that initially implemented this feature #166. Any thoughts on adopting ISO8601?